### ISSUE TITLE
Refresh on directory change

### ISSUE THREAD

**Issue by lonix1** [tmux-plugins/tmux-sidebar#37]
When changing a directory, I need to close and reopen the sidebar.

Is there a way to make it update itself automatically when the directory is changed?

**Comment by vignesh0025** [comment]
I too feel this will be a nice feature. Infact, I downloaded this extension expecting this is the default behavior.

**Comment by lonix1** [comment]
Yes I should have mentioned that... I expected that to be the way it works. I don't see much need for it otherwise - `tree` is easy to use.

**Comment by cwfoo** [comment]
Why was this issue closed? Almost every discussion I have read about this extension mentions this as a problem.

**Comment by lonix1** [comment]
Because the last update was in 2018, it's unmaintained. I'll reopen if it makes you happy.

**Comment by SuperJumpBros** [comment]
I agree with this, hopefully some update comes out evantually.

**Comment by kiyoon** [comment]
I recommend using this plugin with a more modern file explorer. You may want to use the explorer's api to even interact with it.

Check out the demo of [treemux](https://github.com/kiyoon/treemux) that uses Nvim-Tree as explorer and dynamically update the directory. Hopefully this meets your need or inspires you to further improve the plugin.
